src/components/FavouriteBtn.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faStar } from '@fortawesome/free-solid-svg-icons';
+import { increaseStarCount } from '../services/storage';
+
+export default function FavouriteBtn({id}) {
+
+    return (
+        <button
+            type='button'
+            className='card__favourite-wrapper'
+            onClick={() => increaseStarCount(id)}
+        >
+            <FontAwesomeIcon
+                className='card__favourite'
+                icon={faStar}
+            />
+        </button>
+    )
+}
```

Comment:
FavouriteBtn component: here I import increaseStarCount method and I wish to connect all logic from global storage.js to here